RecoMTD/DetLayers/src/MTDDetRing.cc

```diff
@@ -12,7 +12,10 @@
 #include <iostream>
 #include <vector>
 
+//#define EDM_ML_DEBUG
+
 using namespace std;
+const std::string MTDDetRing::metname = "MTD|RecoMTD|RecoMTDDetLayers|MTDDetRing";
```

Comment:
wouldn't this trigger a quadruplicate printout (one for each category)?

Reply:
@slava77 yes, you are correct, it does, but this is what it already was present, and it is used in several other places in the RECO code. Are you suggesting to simplify it? It might make sense to me.

Reply:
yes, please keep it simple (or at least do not increase complexity compared to what it was when using a single category of "MTDDetRing")